packages/config/config/devices/0x015f/mh-ds221.json

```diff
@@ -0,0 +1,195 @@
+{
+	"manufacturer": "McoHome Technology Co., Ltd.",
+	"manufacturerId": "0x015f",
+	"label": "MH-DS221",
+	"description": "Micro Switch",
+	"devices": [
+		{
+			"productType": "0xd221",
+			"productId": "0x7102"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Behavior if power fails",
```

Comment:
You're missing some things from our style guide. We have rules for capitalization and such. Please take a look and correct throughout. https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide